webpack/src/Router/HostReports/IndexPage/constants.js

```diff
@@ -1,8 +1,9 @@
 import { getControllerSearchProps } from 'foremanReact/constants';
 
-export const HOST_REPORTS_SEARCH_PROPS = getControllerSearchProps(
-  'host_reports'
-);
+export const HOST_REPORTS_SEARCH_PROPS = {
+  ...getControllerSearchProps('/host_reports'),
+  controller: 'host_reports',
```

Comment:
Just out of curiosity: why is this change?

Reply:
iirc, the call from the host tab that uses the autocomplete path didn't work, the path got corrupted.
but when using the `/` before `host_reports` it works well for both the index page and the host's puppet tab.